Add two new columns to `connectors` for UI to show/hide properties

travjenkins opened the pull request:
**Description:**

Added two new columns to the `connectors` table. 

https://github.com/estuary/ui/issues/1194 / https://github.com/estuary/flow/issues/1552
`capture_interval`
when `null` the UI will hide the interval input on the Capture edit/create workflow
when a value the UI will show the interval input and default it to the provided value

https://github.com/estuary/ui/issues/1219
`disable_backfill`
when `false` the UI will show backfill
when `true` the UI will hide backfill


**Workflow steps:**

Run a migration on Supabase

**Documentation links affected:**

n/a

**Notes for reviewers:**

Ran `supabase stop` and `supabase start` to make sure the script ran
```
...
Applying migration 48_storage_mapping_directive.sql...
Applying migration 49_tenant_hide_preview.sql...
Applying migration 50_controllers.sql...
Applying migration 51_backfill_controllers.sql...
Applying migration 52_extend_pg_net_timeout.sql...
Applying migration 53_rls_perf.sql...
Applying migration 54_user_info_summary.sql...
Applying migration 55_inferred_schema_triggers.sql...
Applying migration 56_connector_tags_control_flags.sql...
Seeding data supabase/seed.sql...
Started supabase local development setup.
...

```
![image](https://github.com/user-attachments/assets/475db8ce-0212-4218-a4b9-fad53b00d9f9)


<!-- Reviewable:start -->
- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/estuary/flow/1556)
<!-- Reviewable:end -->

psFried reviewed (changes requested):
Unfortunately, I think you've been led astray by our out-of-date docs on migrations. Sorry. The way to do migrations now is to add a new migration and run `alter table` commands to add the columns. You might try using the `pgadmin` workflow from those docs in order to generate the migration, though I'm not sure whether it still works (which is why we switched workflows).

I think we might also want these columns on `connector_tags` instead of connectors. We have things like `autodiscover_interval` on `connector_tags`, and having it there allows us to set it automatically if we ever want to. Would that work from a front-end perspective?

travjenkins commented:
Ahhh - I thought that was how we did it but saw when we added GCP stuff we just updated existing migrations.

Yeah - putting them on tags works for me. I was wondering how we'd want to handle if we updated a connector to make things like interval no longer needed.

psFried reviewed (approved):
LGTM, thanks!
